param/param.go

```diff
@@ -110,6 +111,7 @@ type ValidatorParam struct {
 	OncallInflationWeight          int64      `json:"oncall_inflation_weight"`
 	StandbyInflationWeight         int64      `json:"standby_inflation_weight"`
 	MaxVotedValidators             int64      `json:"max_voted_validators"`
+	SlashLimitation                int64      `json:"slash_limitation"`
```

Comment:
add this field in genesis file: https://github.com/lino-network/lino/blob/master/genesis/upgrade4/genesis.json